package.json

```diff
@@ -14,6 +14,7 @@
     "@react-native-async-storage/async-storage": "1.17.11",
     "async-storage": "0.1.0",
     "graphql": "16.6.0",
+    "lottie-react-native": "^5.1.4",
```

Comment:
Lembrar do `^`. Vc pode automatizar isso: https://medium.com/front-end-weekly/how-to-save-exact-npm-package-versions-7f8ff9a0415a

Reply:
Mas no fim das contas acho que vc nem usou a lib, poderia remover então.